MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h

```diff
@@ -35,15 +35,24 @@ typedef NS_ENUM(NSUInteger, MUXSDKViewOrientation) {
     MUXSDKViewOrientationLandscape
 };
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnullability-completeness"
+
 @protocol MUXSDKPlayDispatchDelegate
 - (void) playbackStartedForPlayer:(NSString *) name;
 - (void) videoChangedForPlayer:(NSString *) name;
 @end
 
+#pragma clang diagnostic pop
+
 @interface MUXSDKPlayerBinding : NSObject {
+
+#pragma clang diagnostic push
```

Comment:
do we still want these in here, or were they for debugging?

Reply:
They're intentional, I needed to annotate nullability for the new methods and avoid accidentally changing the implicit nullability for the existing methods. 

That created a ton of build warnings that are visible when compiling an app with the SDK binary. Worked around them by adding these as escape hatches that suppress just that warning. (`-Wnullability-completeness`)  

As part of v4.0.0 the nullability should be made explicit on everything in the public headers.

Reply:
gotcha, sounds good. Just havent' seen those before